code/modules/antagonists/heretic/items/heretic_blades.dm

```diff
@@ -105,3 +105,12 @@
 	icon_state = "dark_blade"
 	inhand_icon_state = "dark_blade"
 	after_use_message = "The Torn Champion hears your call..."
+
+// Path of Cosmos's blade
+/obj/item/melee/sickly_blade/cosmic
+	name = "\improper cosmic blade"
+	desc = "A blade reflecting the cosmos, made from strange matter. \
+		Containing cosmic radiation, deconstructing organic lifeforms to their building blocks."
```

Comment:
```suggestion
	desc = "A mote of celestial resonance, shaped into a star-woven blade. \
		An iridescent exile, carving radiant trails, desperately seeking unification."
```

Hi, you don't have to take my suggestions particularly seriously but I wanted to suggest a description for the blade, following the previous patterns I had used for the other blades I've worked on.

Reply:
Sure why not